### PR TITLE
Update tld list and allow build with newer ghc

### PR DESCRIPTION
**Pull request by gavwhela**
Let me know if any changes to this merge would be preferable.

### CODE REVIEW

File: process.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```
Comment (jprider63) [inline process.py]: Can you drop this change? I don't have `python2` as an executable on my system.

**Review by jprider63 — COMMENTED** [review]
Looks good otherwise. Once you make that update, I'll merge and deploy to hackage.

**Comment by gavwhela** [comment]
I've updated process.py to be compatible with both python2 and python3.

**Comment by jprider63** [comment]
Thanks! Uploaded to hackage.

**Pull request merged**